vnext/ReactWindowsCore/Modules/WebSocketModule.h

```diff
@@ -16,23 +16,24 @@ class WebSocketModule : public facebook::xplat::module::CxxModule {
  public:
   enum MethodId { Connect = 0, Close = 1, Send = 2, SendBinary = 3, Ping = 4, SIZE = 5 };
 
-  WebSocketModule();
+  WebSocketModule(
+      std::function<std::shared_ptr<IWebSocketResource>(const std::string &, bool, bool)> &&resourceFactory = nullptr);
```

Comment:
>resourceFactory [](start = 92, length = 15)

(nit) Since this is needed only for test, it could be cleaner just to create a separate "SetResourceFactory(...)" method, instead of having optional parameter in constructor. Either way works though.